src/languages/xml.js

```diff
@@ -54,6 +58,10 @@ function(hljs) {
         begin: '<\\!\\[CDATA\\[', end: '\\]\\]>',
         relevance: 10
       },
+      {
+        className: 'symbol',
+        begin: '&[a-z]+;|&#[0-9]+;|&#x[a-f0-9]+;', relevance: 10,
```

Comment:
This probably should not be relevance 10, many other formats that are HTML variants also have entities... This might be why your relevance tests are breaking.

Reply:
I slightly modified the highlighting of entities. In the new version, entities are already highlighted in the attribute strings.